src/openzaak/components/autorisaties/utils.py

```diff
@@ -101,7 +101,7 @@ def send_applicatie_changed_notification(
 ):
     from openzaak.utils import build_fake_request
 
-    viewset = ApplicatieViewSet()
+    viewset = ApplicatieViewSet(request=None)
```

Comment:
Is it necessary? I thought `request` is optional

Reply:
The change in notifications-api-common now requires it to be set: https://github.com/maykinmedia/notifications-api-common/pull/14/files#diff-c2e62ddfe4c65f18ec4a1121e116a4fad306ba4ccb8cb738b5b61eae2520784dR140

Reply:
I'll see if I can change it to `getattr` so that this line can be reverted